lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -25,6 +26,10 @@
 #include "llvm/ADT/ScopedHashTable.h"
 #include "llvm/ADT/SmallPtrSet.h"
 #include "llvm/ADT/StringExtras.h"
+#include "llvm/ADT/StringMap.h"
+#include "llvm/ADT/StringSwitch.h"
+#include "llvm/Support/Error.h"
+#include "llvm/Support/JSON.h"
```

Comment:
Please prune unneeded #includes.  StringMap comes in from FIRAnnotations, StringSwitch.h isn't used, Error.h may already come in, etc.

Reply:
Cleaned up in a `!fixup` commit. There are a few more extraneous ones in this file that I didn't add that I can rip out on the `main` branch.